### PR TITLE
Change en_US to en_UK

### PR DESCRIPTION
**Pull request by cefoo**
Changes US spellings to UK spellings.
Reviews and fixes paths.
Reviews and fixes typos.

### CODE REVIEW

**Comment by bittlingmayer** [comment]
My main comment here is:

Always "move" files when renaming (`git mv` on the command line), do *not* delete and recreate the new file.

The reason is that the editing history gets lost, so people's contributions gets lost.

In this case luckily it doesn't matter much.

File: events/zurich-9.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 parent: Events
 title: Machine Translation Meetup
-description: Machine Translation Meetup (MTM) - Zurich 
+description: Machine Translation Meetup (MTM) - Zürich
```
Comment (bittlingmayer) [inline events/zurich-9.md:4]: IMHO it's just a weird bug in Meetup that they use the ü in English, it's not correct or official (e.g. on the English version of the cities official website).

File: people/philipp-koehn.md [hunk 2]
```diff
@@ -8,14 +8,14 @@ description: Machine translation researcher
...
 
 His early contributions include:
 
 * the Moses machine translation system
 * the Europarl corpus
-* organizing the machine translation research community, including WMT
+* organizing the machine translation research community, including [WMT](/community/communities.md#Workshop_on_Statistical_Machine_Translation)
 
-He was a professor at the University of Edinburgh and then Johns Hopkins University and Chief Scientist at [Omniscien Technologies](industry/companies.md#omniscien-technologies).
+He was a professor at the University of Edinburgh and then Johns Hopkins University and Chief Scientist at [Omniscien Technologies](industry/companies.md#Omniscien_Technologies).
```
Comment (bittlingmayer) [inline people/philipp-koehn.md:19]: Lowercase was correct

File: people/philipp-koehn.md [hunk 3]
```diff
@@ -8,14 +8,14 @@ description: Machine translation researcher
...
 In 2003, while at the University of Southern California, he co-authored a paper on phrase-based statistical machine translation,
-along with [Franz Josef Och](people/franz-josef-och.md) and [Daniel Marcu](people/daniel-marcu.md).
+along with [Franz Josef Och](franz-josef-och.md) and [Daniel Marcu](daniel-marcu.md).
 
 His early contributions include:
 
 * the Moses machine translation system
 * the Europarl corpus
-* organizing the machine translation research community, including WMT
+* organizing the machine translation research community, including [WMT](/community/communities.md#Workshop_on_Statistical_Machine_Translation)
```
Comment (bittlingmayer) [inline people/philipp-koehn.md:17]: Lowercase was correct

**Review by bittlingmayer — APPROVED** [review]
Did you use some sort of tool?

**Comment by cefoo** [comment]
> My main comment here is:
> 
> Always "move" files when renaming (`git mv` on the command line), do _not_ delete and recreate the new file.
> 
> The reason is that the editing history gets lost, so people's contributions gets lost.
> 
> In this case luckily it doesn't matter much.

Thanks for the comment, I just named the files in Atom, which I now see has more implications that I realized.

**Comment by cefoo** [comment]
> Did you use some sort of tool?

Not really, just Language Tool and spellcheckers. I relied heavily on Cambridge dictionary.

**Pull request merged**